src/rust/include/librustzcash.h

```diff
@@ -8,7 +8,16 @@
   #include <stdalign.h>
 #endif
 
-#define ENTRY_SERIALIZED_LENGTH 180
+#define NODE_SERIALIZED_LENGTH 171
+#define ENTRY_SERIALIZED_LENGTH NODE_SERIALIZED_LENGTH + 9
```

Comment:
```suggestion
#define ENTRY_SERIALIZED_LENGTH (NODE_SERIALIZED_LENGTH + 9)
 ```

Macros that expand to an expression that is not a single token should always put parens around that expression. Consider the expansion of `ENTRY_SERIALIZED_LENGTH*2`, for example.